.infrastructure/bootstrap.sh

```
@@ -0,0 +1,11 @@
+#!/bin/bash
```

Add a safety flag so the script fails fast on errors. Consider adding at the top (immediately after the shebang): `set -euo pipefail` or at least `set -e`. This prevents the script from continuing after an apply fails and makes debugging easier.